libs/feature/map/src/lib/utils/map-utils.service.ts

```diff
@@ -137,13 +136,7 @@ export class MapUtilsService {
         )
       )
     } else if (layer && layer.type === 'wms') {
-      geographicExtent = from(
-        new WmsEndpoint(this.proxy.getProxiedUrl(layer.url))
-          .isReady()
-          .then((endpoint) => endpoint.getLayerByName(layer.name))
-      ).pipe(
-        map((layer: any) => layer.boundingBoxes['EPSG:4326']) // eslint-disable-line -- ogc-client lacks typing here
-      )
+      geographicExtent = this.wmsUtils.getLayerLonLatBBox(layer)
```

Comment:
Regarding my other comment, this could become:

```ts
from(
        new WmsEndpoint(this.proxy.getProxiedUrl(layer.url))
          .isReady()
          .then((endpoint) => endpoint.getLayerByName(layer.name))
          .then((layer) => layer.getLonLatBoundingBox())
      )
```